jdbc/src/main/java/com/interface21/jdbc/core/AugumentsPreparedStatementSetter.java

```diff
@@ -3,11 +3,11 @@
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
 
-public class DefaultParameterSetter implements ParameterSetter {
+public class AugumentsPreparedStatementSetter implements ParameterSetter {
```

Comment:
제가 코멘트에서 오타를.. 냈었는데 복붙하셨나보네요 😢 
```suggestion
public class ArgumentsPreparedStatementSetter implements ParameterSetter {
```